src/content/docs/mobile-apps/dashboards-new-relic-mobile-apps.mdx

```diff
@@ -0,0 +1,116 @@
+
+### Filter using the chart legend
+
+Tap on a legend in any chart with legends to see that series only, and remove the rest of the series from the chart. This helps you isolate the data you want to analyze.
+
+### Adjust time settings with **Time Picker**
+
+By default, each chart in the dashboard shows data for the time period specified when they were created in the query builder. However, you can use the time picker to change the time range of the data on display and set the same range for all charts. This is particularly useful while troubleshooting incidents, if you need to narrow down your data to observe what happened in a specific time period.
+
+To change the time range:
+
+Tap the clock icon at the top right of your dashboard
+Choose one of the available options from the menu by tapping radio button
+Or customize the time range with specific start and end timestamps using the custom menu.
+
+### Share your data
+
+1. In your dashboard, tap the Share icon in the top right
+2. Use your device’s share sheet to send your dashboard link
+
+### Data Scrubber
+
+The chart scrubber helps you select a data point or facet in a chart when the chart is too crowded and facets are too close to each other.
+
+Move your finger along the needle to smoothly select the adjacent facets and view their associated data points. You can also lighten a heavily populated chart by unselecting one or more of the attributes that appear in the UI.
+
+### Template variables: dynamically filter dashboards
+
+In New Relic dashboards, template variables are placeholders in the NRQL queries that can be later assigned a value, which allows for more flexibility when handling different use cases. Once you create dashboards with these variables in New Relic on the web, you can then view them in New Relic Mobile, allowing you to easily filter through your dashboard for more precise data insights.
+
+For more information on creating template variables on the web, read our [Dashboard docs](/docs/query-your-data/explore-query-data/dashboards/dashboard-template-variables/) 
+
+After you create your dashboard on the web, you can see that same dashboard and filter by those same variables in the mobile app:
+
+<img
+    title="Web view of dashboards"
+    alt="A screenshot depicting dashboards on the web before they're ready for mobile"
+    src={dashboardsDashboardstoPrepareforMobile}
+/>
+
+<figcaption>
+New Relic dashboards on the web.
+</figcaption>
+
+<img
+    title="New Relic mobile dashboards"
+    alt="A screenshot depicting New Relic's mobile dashboards"
+    src={dashboardsMobileDashboards}
+/>
+
+<figcaption>
+New Relic dashboards for mobile can be filtered by the same variables as on the web.
+</figcaption>
+
+<img
+    title="mobile dashboards filtered by variable"
+    alt="A screenshot showing variable filters for New Relic mobile dashboards."
+    src={dashboardsFilterMobileDashboardsbyVariable}
+/>
```

Comment:
The 2 last screenshots display too big, in my opinion. I'd add width="80%;", for example, or 70%.

Reply:
good call! Thanks @nbaenam